tasks/Task.cpp

```diff
@@ -57,18 +57,6 @@ void Task::processIO()
 
 }
 
-bool Task::setLed_list(std::vector<ledlamp_bowtech::LedLamp > const& ledlist)
-{
-    for (size_t i = 0; i < ledlist.size(); ++i)
-    {
-    	lamps.setPowerUpLightLevel(ledlist[i].power_up_light_level, ledlist[i].address);
-    	usleep(100000);
-    	lamps.setLightLevel(ledlist[i].light_level, ledlist[i].address);
-    	usleep(100000);
-    }
-
-    return(ledlamp_bowtech::TaskBase::setLed_list(ledlist));
-}
```

Comment:
I would go for a std::vector of float (where 0 is off and 1 is on), and require that the vector's length is the same than the list of lamps configured in the properties.